### ISSUE TITLE
Add swagger annotations to controllers

### ISSUE THREAD

**Issue by cdparra** [socialappslab/appcivist-platform#65]
Following examples in controllers.Users and [Swagger's api spec](https://github.com/swagger-api/swagger-core/wiki/Annotations), add annotations for the remaining controllers. 

In this way, the endpoint /api/doc will automatically display documentation based on our API endpoints and classes (check the end result at [appcivist heroku test site](https://appcivist-pb.herokuapp.com/api/doc))

After adding Swagger annotations to a controller, make sure to add the proper documentation route for that controller as shown in the following example: 

```
GET       /api/doc.json/campaign                        @pl.matisoft.swagger.ApiHelpController.getResource(path="/campaign")
```

**Comment by cdparra** [comment]
@Animesh, I think if you work on this, it will also allow you to review the current API design and make suggestions about what should be removed/add/simplified

**Comment by animeshpathak** [comment]
So, all of them at https://github.com/socialappslab/appcivist-platform/tree/master/app/controllers , right?

**Comment by cdparra** [comment]
Yes, all controllers are in the same package. And to know exactly what method is related to each endpoint, you can checkout the "routes" file in the root folder of the repository
